packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -17,11 +17,13 @@ import MarkdownPostContent from './MarkdownPostContent';
 import { SquadPostWidgets } from './SquadPostWidgets';
 import { isSourcePublicSquad } from '../../graphql/squads';
 import { useAuthContext } from '../../contexts/AuthContext';
+import ShareYouTubeContent from './ShareYouTubeContent';
 
 const ContentMap = {
   [PostType.Freeform]: MarkdownPostContent,
   [PostType.Welcome]: MarkdownPostContent,
   [PostType.Share]: SharePostContent,
+  [PostType.VideoYouTube]: ShareYouTubeContent,
```

Comment:
Can't we simply reuse the article here as it's pretty much the same?

Reply:
This is for Shared post with a type of Video. The article post is reused when it came from a direct source.

Reply:
This will actually never evaluate.
The share post can't be directly video, it has to be Share itself.
The subtype will be video:youtube that's being shared.